src/niscope/metadata/functions_addon.py

```diff
@@ -0,0 +1,85 @@
+# TODO(texasaggie97) update this file with all NI-FGEN function information
+
+# These dictionaries are applied to the generated functions dictionary at build time
+# Any changes to the API should be made here. functions.py is code generated
+
+# By default all functions in functions.py will be generated as a public function
+# This will override that with private - add '_' to the beginning of the name, or
+# don't generate at all
+functions_codegen_method = {
+    'InitWithOptions':                 { 'codegen_method': 'private',  },
+    'InitiateAcquisition':             { 'codegen_method': 'private',  },
+    'close':                           { 'codegen_method': 'private',  },
+    'Abort':                           { 'codegen_method': 'private',  },
+    '.etAttribute.+':                  { 'codegen_method': 'private',  },  # All Set/Get Attribute functions are private
+    'init':                            { 'codegen_method': 'no',       },
+    'error_message':                   { 'codegen_method': 'private',  },
+    'GetError':                        { 'codegen_method': 'private',  },
+    'ClearError':                      { 'codegen_method': 'no',       },
+    'LockSession':                     { 'codegen_method': 'no',       },
+    'UnlockSession':                   { 'codegen_method': 'no',       },
+    '.+ExtCal':                        { 'codegen_method': 'no',       },  # External Calibration is not supported by the Python API
+    'CalAdjust.+':                     { 'codegen_method': 'no',       },  # External Calibration is not supported by the Python API
+    '.+UserDefined.+':                 { 'codegen_method': 'no',       },
+    'SetAttributeViSession':           { 'codegen_method': 'no',       },
+    'GetAttributeViSession':           { 'codegen_method': 'no',       },
+    'GetNextInterchangeWarning':       { 'codegen_method': 'no',       },  # Not applicable to Python API
+    'ResetInterchangeCheck':           { 'codegen_method': 'no',       },  # Not applicable to Python API
+    'ClearInterchangeWarnings':        { 'codegen_method': 'no',       },  # Not applicable to Python API
+    'GetNextCoercionRecord':           { 'codegen_method': 'no',       },  # Not applicable to Python API
+    'error_query':                     { 'codegen_method': 'no',       },
+    'revision_query':                  { 'codegen_method': 'no',       },
+    'SampleMode':                      { 'codegen_method': 'no',       },  # Equivalent attribute is available
+    'GetNormalizationCoefficients':    { 'codegen_method': 'no',       },  # Has void param
+    'GetScalingCoefficients':          { 'codegen_method': 'no',       },  # Has void param
+    'Fetch':                           { 'codegen_method': 'no',       },  # Has niScope_wfmInfo param
```

Comment:
point to an issue for handling this to make clear it's temporary

Reply:
Added